### PR TITLE
[WIP] Update README.md

### PR DESCRIPTION
**Pull request by nooperation**

### CODE REVIEW

**Comment by bill-auger** [comment]
im a bit confused as to what is happening with this repo - this github organization has 5 developers listed - on bitbucket cinderblocks is alone - is anyone listed as a developer in this organization planning to contribute to the new fork? or is the idea to just let this repo go stale?

it should be considered that on github this project has 20 watchers and 40 forks and on bitbucket only 9 watchers and 3 forks - it also should be considered that having orphaned base forks scattered about the internet is very confusing to users especially when they all appear to be "official" - there is unfortunately already one such stale base fork on bitbucket https://bitbucket.org/radegastdev/ and even more unfortunately that is where the radegast website directs users to report bugs - also the official radegast wiki directs users to this repo for the source code 

if the intention of this PR is to migrate everyone to bitbucket then i would hope that this entire organization is destroyed after some reasonable time - if that is not the plan then i suggest that the appropriate thing to do here is merging in the changes from cinderblocks bitbucket fork periodically in order to keep this repo viable and open for reporting issues - and of course someone should be attending to those issues or at the very least forwarding them to cinderblocks's fork - if no one on the github organization wants to do maintain this fork then perhaps cinderblocks cuold be added as an admin here - or else you really should just destroy the github presence entirely

**Comment by nooperation** [comment]
The main problem is that I don't know anyone who has admin access to this project on github so I'm really not sure what to do about it. The other problem is that the original radegast site ( http://radegast.org/wp/ ) where releases are kept is also without an owner so nobody can even make releases in its current state.

I saw that the Radegast entry page on http://wiki.secondlife.com/wiki/Third_Party_Viewer_Directory/Radegast has been changed to link to Cinder's bitbucket as the official repository and https://radegast.life/ as the official homepage, which is part of the reason behind the PR.

I'll set this to WIP because I don't want it to be accidentally commited until a better solution is found

**Comment by bill-auger** [comment]
this is a very unfortunate situation - i went through the same thing about 6 months ago with the freewheeling project - the only person with any access to the sourceforge repo could only update the static website to say "moved to github" but he was not able to update the sourceforge site in any way to indicate that the project had to move and is not simply abandonware - at least in that case the sourceforge repo was the only one that was abandoned - in this case there are at least three - i forgot to mention he had another repo on sourceforge https://sourceforge.net/projects/radegast/ - that one has one other team member - i just messaged that person most of the same questions i raised here

there has been ample time for Latif to "pass the torch" to another maintainer since the three years ago when he first asked someone to take over - i am inclined to presume that means that Latif did not endorse cinderblock's fork but i quite suspect that simply no one asked about doing a proper "official" migration

does anyone know how to contact Latif's family to ask if perhaps they could help in some way?

the bottom line tho is that whoever has permissions to accept this PR could maintain this repo by periodically merging cinderblock's changes and monitoring issues - if no one wants to do that then at the very least you should set an auto-response on new issues directing them to report the issue on bitbucket instead

**Comment by cinderblocks** [comment]
Hello,

I was unable to find anyone with admin access for this team on GitHub either. That goes for libopenmetaverse, the Radegast JIRA, radegast.org, and otherwise. I have reached out as best as I could, but it doesn't seem that Latif left any of the "keys" behind. Note: I have not attempted to contact his family as I do not know the social rules are for what an adequate period of grieving would be.

> there has been ample time for Latif to "pass the torch" to another maintainer since the three
> years ago when he first asked someone to take over - i am inclined to presume that means 
> that Latif did not endorse cinderblock's fork but i quite suspect that simply no one asked about
> doing a proper "official" migration

Radegast-ng did not exist until a while after Latif's passing. I had been biding time waiting to see if someone else had been "passed the torch." This doesn't seem to have happened. It was brought to my attention that voice was no longer working in Radegast as well as issues with screenreader, and this left those with accessibility needs unable to enjoy virtual worlds. It was at that point that I forked. I had meant to ask Latif about taking over development, but it always felt like an awkward time to bring it up as you can imagine and his passing was very sudden (to us on the internet) although he had been ill for quite some time. Suffice it to say, Latif and I mutually respected each other, and I am sure he would not have anything against my fork. 

The domains have begun expiring (as an example, the Donate link on radegast.org's main page now makes several redirects until landing on a page serving malware,) and it is only a matter of time before the rest of the infrastructure goes offline since there is no one to pay for and maintain it.

Ideally, Radegast-ng could be reconciled back into this repository and someone could clear up the issue of copyright (as far as I'm aware, Radegast Development Team is not a legal entity) but I'm not holding my breath.

**Comment by bill-auger** [comment]
what is the copyright issue exactly?

surely the concern is not that "Radegast Development Team" may not be a legal entity - the same concern would equally apply to "Cinderblocks Design Co" - but that is not an problem - as i understand it, literal copyright statements are actually optional - in most nations one acquires copyright implicitly at the moment a work is published publicly unless they dedicate it to the public domain - the literal copyright statement is only informational - if no such legal entity exists named "Radegast Development Team", this is conventionally interpreted as "everyone who has contributed substantially to the work" - in the bad old days before git that meant the contents of the AUTHORS file but with git attributions are more precise in the commit log

in case it is important, you can easily get that list of unique contributors with this command:
```
$ git log --pretty="%an %ae" | sort | uniq
```

according to github those correspond to these github logins:
```
@lkalif
@logicmoo
@nooperation
@robincornelius
@ole1986
@RevolutionSmythe
@SignpostMarv
@fly-man-
@takeshich
@eumario
@CmdrCupcake
@elliena-bulmer
```

on first look there is only one flag raised - that is the commits by 'root' - but this was probably 'ole1986' in the following commits:
```
$ git log --pretty="%h %an %ae" | grep root
2eb846f ole1986 root@CentOS-65-64-minimal.(none)
d3a648c root root@CentOS-65-64-minimal.(none)
```

if it were me, i would purge and re-implement commit d3a648c and any others like it that can not be plainly attributed

a much bigger concern IMHO is the licensing of the many blobs and of the program itself - the BSD 3-clause prohibits anyone from using the radegast name in derived forks - it would probably be reasonable to consider this repo, the sourceforge repo, and the radegastdev repo on bitbucket to be official because they were established by Latif - but the fact that no one ever asked Latif to endorse a fork would make any forks unofficial - and so if i were to fork now, i would be compelled to change the name entirely unless at least one of the official repos remains continuously maintained and in sync with mine as the new upstream - if ever all official repos go abandoned at any time in the future then clause 3 casts it's shadow over any other forks - in other words i dont think it is wise to use the name unless cinderblocks can be added to either the github team or the sourceforge team because i have not yet seen anyone say "yes, we will maintain the github repo for as long as cinderblocks maintains the bitbucket fork" - so the name may need to change someday anyways - @cinderblocks if you know how to contact the family, you really should do that without delay

as fr the blobs, some of them i was not able to find a license or source code for anywhere on the internet - i did not look exhaustively but i should not have needed to either - this information should (must) be provided along with all distributions - this is another concern i would be put a priority on - and i mean immediately, before i wrote another LOC - i will help with this if you like

**Comment by cinderblocks** [comment]
> surely the concern is not that "Radegast Development Team" may not be a legal entity - the 
> same concern would equally apply to "Cinderblocks Design Co"

Cinderblocks Design is a subsidy of Ultimate Gaming Europe AB. Thanks.

> if ever all official repos go abandoned at any time in the future then clause 3 casts it's shadow
> over any other forks

No it doesn't. It's an advertising clause.

> as fr the blobs

Radegast-ng doesn't use them, and it isn't really applicable to this PR.

**Comment by bill-auger** [comment]
i raised the issue of what is or isn't a legal entity only to make the point that the name on the copyright line is not required to be legit because i suspected that was the concern you wanted clarification about

this PR is clearly not only about edits to the README file - i think the intention was as the last chance effort to spark a conversation as to the future of this repo and by extension, the future of the radegast program itself - this is why i am raising all of these issues here - if there is some other particular concern regarding copyright or anything else please do mention it here for the benefit of the community

by "blobs" i mean to say: any binary files that were not compiled from any source code included in the project itself, such as the many .dll files in the radegast sources and also any packages pulled in from nuget - i know you are aware of these type of files because there is at least one commit where you added some of them with message "Add x86_64 fmodstudio blobs" - this is not so much about the sources but that all of the declared licenses (LGPL, BSD, MIT, and Apache2) require that their license files be present in all copies, source or binary - again i havent looked exhaustively but the only actual license i have found is the BSD 3-clause of radegast itself - to be clear, the nuget packages are not exempt for this either - if they are not distributing the license files in their binary packages (i could not find any) then they are violating those licenses and that makes any program using those packages illegitimate

another issue i found already is that the radegast license lists the AIMLBot library as LGPL, probably because the XML data files are LGPL, but the actual sources that built the "AIMLbot.dll" binary are clearly AGPL https://sourceforge.net/p/aimlbot/code/HEAD/tree/trunk/Docs/License.txt - when any program links with AGPL code, the entire program can only be distributed under the AGPL - this has multiple implications - one is that all source code must be available and all licenses in the program must be compatible with the GPL - most notably, the FMOD library is incompatible because of it commercial restriction so that both AIMLBot and FMOD can not be distributed in the same program - so far none of these issues are critical - they have resolutions but resolving these issues should be a priority - regarding the AIMLBot and FMOD issue, that one's a no-brainer - just ditch the AI bot, probably no one uses it anyways

i was able to find the sources for all of the libraries mentioned in the README except for 'primmesher' - the source code for this seems not to be available anywhere on the internet - although the BSD license that is attributed to it does not require the availability of source code i would not be comfortable with using it if only because it is otherwise not possible to verify that it actually had the BSD license - so that is something i would want to look into - the opensimulator.org site still has links scattered about to it's forge site but they have been broken for some time - perhaps someone there could produce the sources

these are just the issues i discovered in a short time and i would expect to find more upon a deeper look - in any case there are clear licensing issues and if youd like to make everything fully legit i am offering to help - i could even set up a public instance of fossology if others want to help

**Comment by fennectech** [comment]
This viewer needs to continue    I dont want it to die!   radegast is really the only good text viewer for linux/windows

**Comment by fly-man-** [comment]
I will leave my 2 cents on this PR as well. 

Last I checked there's 13 people who worked on Radegast including myself with some patches. However @cinderblocks her repo at the moment is the most up to date one and seems to have voice and other pieces working.

Since she's also the person behind LibreMetaverse, making sure the LibOMV doesnt bleed dead as well, I see no issues putting this in the README and point people to it.

**Comment by bill-auger** [comment]
the issue is not so much which repo is considered to be the canonical master repo - git is de-centralized by nature and the concept of a canonical master repo does not apply - the important concern to address here is that this repo on github does in fact exist and it will continue to exist unless someone deletes it - that is not itself any problem - the only problem with that is that it's code has fallen behind the bitbucket repo and no one is monitoring the issue tracker

rather than simply mentioning in the README that this repo is "dead" and the bitbucket repo is the only "official" one; a far better solution is to make cinderblocks an admin of this repo in order to keep the code up to date and accept bug reports and pull requests from github users and bitbucket users, or at least for someone who can already do those thing to just do them - this is not a hugely popular program - it should be very easy to maintain this repo

it would make sense to direct everyone to bitbucket only if it were absolutely impossible to maintain this github repo - that may be unfortunately the case for the sourceforge repo and the original bitbucket repo; but this one is still viable - the very fact that this PR may be merged proves that - there is no good reason to direct people to bitbucket unless the github issue tracker and pull requests were disabled; but even if that is possible, i dont see any good reason to do that either - keeping the code up to date is very easy to do and accepting bug reports and pull requests is no more difficult to do on one website, or on two, or on a mailing list, or through any other channels

at the very least, if the code is on github, then someone should keep it up to date - anyone with access could be doing that now - it takes only a few seconds of someone's time once per month or so - if no one wants to keep the code up to date, or if the issue-tracker/pull-requests can not be disabled, then it would be best if the code here was deleted entirely, leaving nothing but the "farewell" README, as not to entice anyone into using the issue tracker or sending a pull request

when last we left this discussion, it was concluded that the main problem was that latif left no one with admin access and that it had not been an "adequate period of grieving" to contact his family and ask if they could do anything to help - so that was a year ago now - surely that adequate period of grieving has passed - if anyone knows how to contact latif's family that would surely be worth trying
